apps/pixeloven/site/docs/getting-started/generator.mdx

```diff
@@ -0,0 +1,50 @@
+---
+id: generator
+title: Pixeloven Generators
+---
+
+Creating new apps, packages, or components within your project doesn't have to be a chore, we've provided generators to lend a hand.
+
+> Note: This section is a work in progress and should contain basic information for creating and working with generators
+
+## Overview
+
+To start any of the generators run the `generate` command
+
+```
+yarn pixeloven generate
+```
+
+The generators are somewhat opinionated and make a few assumptions about the organization of your poject. See the instructions for each individual generator below for specifics.
+
+## Applications
+
+1. To start a new app, run the `generate` command from the root of your project and choose `App` from the selection menu.
+1.
+
+## Packages
```

Comment:
I suggest re-ordering these sections based on how common it is for a user to have to create a package vs a new component.

Reply:
oh good point, we should reorder the generator as well

Reply:
done

Reply:
Looks like this is resolved now